metafix-runner/build.gradle

```diff
@@ -30,6 +30,9 @@ dependencies {
   implementation "org.metafacture:metafacture-xml:${versions.metafacture}"
   implementation "org.metafacture:metafacture-yaml:${versions.metafacture}"
   implementation "org.metafacture:metamorph:${versions.metafacture}"
+  runtimeOnly 'org.apache.logging.log4j:log4j-core'
+  implementation 'org.apache.logging.log4j:log4j-1.2-api:2.9.1'
+  implementation 'ch.qos.logback:logback-classic:1.3.14'
```

Comment:
```suggestion
  runtimeOnly "org.slf4j:slf4j-log4j12:${versions.slf4j}"
```

Reply:
With this suggestion, I get no logging output, and the same `Class path contains multiple SLF4J bindings.` warning that I get without this PR.

Reply:
That warning aside (different issue, IMO), you would have to increase the log level to `DEBUG`.

Reply:
Same with `DEBUG` in `log4j.xml`: no logging output. As you wrote below, it must be use some other config?

Reply:
That's not what I saw. Changing the log level (priority) in `log4j.xml` and then running `./gradlew installDist` yields debug output with `log-object`.

Reply:
Weird, I have `<priority value="DEBUG" />` in `metafix/src/main/resources/log4j.xml`, only `runtimeOnly "org.slf4j:slf4j-log4j12:${versions.slf4j}"` added in `metafix-runner/build.gradle`: no logging output for `log-object`.

Reply:
And you ran `./gradlew installDist` afterwards? For me on `master`:

1. Copy `log4j.xml` from metafacture-core to `metafix/src/main/resources/`.
2. Change priority `info` to `DEBUG` in `metafix/src/main/resources/log4j.xml`.
3. Add `runtimeOnly "org.slf4j:slf4j-log4j12:${versions.slf4j}"` to `metafix-runner/build.gradle`.
4. Run `./gradlew installDist`.
5. Change into `metafix-runner/build/install/metafix-runner` directory.
6. Run `bin/metafix-runner /path/to/flux-with-log-object`.
7. See "DEBUG [main] ..." output.

Reply:
@blackwinter your proposal is working for me

Reply:
I just tried it on my home machine, and here it works. Maybe some cached dependency bringing in some other config file on my office machine? So that would be +1 for using only `runtimeOnly "org.slf4j:slf4j-log4j12:${versions.slf4j}"`. However, it works here even without adding any dependency, just by switching `INFO` to `DEBUG` in `metafix/src/main/resources/log4j.xml`.

Reply:
> However, it works here even without adding any dependency

Indeed. Probably because it's pulled in by metafacture-runner.So let's just drop it altogether then.